lib/Chat/Parser/UserMention.php

```diff
@@ -90,8 +90,14 @@ protected function parseMessage(Message $chatMessage): void {
 			return mb_strlen($m2['id']) <=> mb_strlen($m1['id']);
 		});
 
+		$actor = $this->participantService->getParticipantByActor($chatMessage->getRoom(), $chatMessage->getActorType(), $chatMessage->getActorId());
```

Comment:
Since this does a DB query we should only do it when we found a `@all` message and we don't know the info for the actor yet.
But now I got an idea how we can "fix it"